bigtable/google/cloud/bigtable/table.py

```diff
@@ -138,6 +139,73 @@ def name(self):
             project=project, instance=instance_id, table=self.table_id
         )
 
+    def get_iam_policy(self):
+        """Gets the access control policy for a table.
+
+        For example:
+
+        .. literalinclude:: snippets_table.py
+            :start-after: [START bigtable_table_get_iam_policy]
+            :end-before: [END bigtable_table_get_iam_policy]
+
+        :rtype: :class:`google.cloud.bigtable.policy.Policy`
+        :returns: The current IAM policy of this table.
+        """
+        table_client = self._instance._client.table_admin_client
+        resp = table_client.get_iam_policy(resource=self.name)
```

Comment:
this sample takes a resource, but this function doesn't.

Reply:
If I clearly understand the logic, this method creates a call of the `get_iam_policy` method from the `bigtable_table_admin_client`. That is why it is taking the `resource` as an argument. 

![image](https://user-images.githubusercontent.com/46078689/69414969-d0208f80-0d24-11ea-95e7-dfa32d40d5a2.png)